src/main/java/org/team199/robot2022/subsystems/Climber.java

```diff
@@ -19,44 +22,57 @@ public class Climber extends SubsystemBase {
     private static final double kNEOFreeSpeedRPM = 5680;
     private static final double kDiameterIn = 1;
     private static final double kDesiredRetractSpeedInps = 1;
-    private static final double kDesiredExtendSpeedInps = 24*.6;
-
-    // Torque is 2 * 9 * 0.5 = 9
-    // Torque on the motor is Torque / ( gearing = 9 ) = 1
-
-    private static final double kVoltsToCounterTorque = 10.5;
-
+    private static final double kDesiredExtendSpeedInps = 6;
     private static final boolean leftInverted = false;
 
-    private static final double extendPositionLeft = 6.315;
-    private static final double extendPositionRight = 6.315;
-
-    private static final double retractPositionLeft = -0.6;
-    private static final double retractPositionRight = -0.6;
     private static final double gearing = 9;
     private static final double kInPerSec = ((kNEOFreeSpeedRPM / gearing) * Math.PI * kDiameterIn / 60);
-    private static double kRetractSpeed = -((kDesiredRetractSpeedInps / kInPerSec)
-            + (kVoltsToCounterTorque / 12)); // ~ -0.06151
-    private static double kExtendSpeed = (kDesiredExtendSpeedInps / kInPerSec); // ~0.30261
-
-    private static final double kSlowDesiredRetractSpeedInps = 2;
-    private static final double kSlowDesiredExtendSpeedInps = 2;
+    private static final double kSlowDesiredRetractSpeedInps = 1;
+    private static final double kSlowDesiredExtendSpeedInps = 1;
 
     // Torque is 2 * 9 * 0.5 = 9
     // Torque on the motor is Torque / ( gearing = 9 ) = 1
-
+    private static final double kVoltsToCounterTorque = 10.5;
     private static final double kSlowVoltsToCounterTorque = (1.1D / 32) * 12;
-    private static final double kSlowRetractSpeed = -((kSlowDesiredRetractSpeedInps / kInPerSec)
-            + (kSlowVoltsToCounterTorque / 12)); // ~ -0.06151
-    private static final double kSlowExtendSpeed = (kSlowDesiredExtendSpeedInps / kInPerSec); // ~0.30261
+
+    private boolean keepPosition = true;
+    private double holdTolerance = 0.05;
+
+    public static final int leftMotor = -1;//because someone requested this and there's no point slapping them in an enum
+    public static final int bothMotors = 0;
+    public static final int rightMotor = 1;
```

Comment:
```suggestion
    public static final int leftMotor = 0;//because someone requested this and there's no point slapping them in an enum
    public static final int bothMotors = 1;
    public static final int rightMotor = 2;
```
Make it so that the motors can be passed directly as array indexes